src/Services/AquiferService.cs

```diff
@@ -78,16 +96,104 @@ public async Task<string> GetContent(string contentid, string? type)
         type ??= "0";
         return await DoApiCall($"resources/{contentid}", ("contentTextType", type));
     }
-    public async Task<string> Post(AquiferItem [] content)
+
+    public async Task<string> Post(AquiferPost post)
     {
+        List<int> mediaids = new();
+        List<int> srids = new();
         //List<AquiferItem>? items = JsonSerializer.Deserialize<List<AquiferItem>>(content);
         string info = "done";
+        Passage? passage = DbContext.PassagesData.Where(p => p.Id == post.PassageId).FirstOrDefault();
+        int? sectionId = (post.SectionId ?? passage?.SectionId) ?? throw new Exception("No SectionId");
+        string fp = HttpContext != null ? HttpContext.GetFP() ?? "" : "";
+        HttpContext?.SetFP("aquifer");
 
-        for (int ix = 0; ix < content?.Length; ix++) {
-            AquiferItem c = content[ix];
-            Console.WriteLine(c);
+        Section? section = DbContext.SectionsData.Where(s => s.Id == sectionId).FirstOrDefault();
+        Artifacttype? artifacttype = DbContext.Artifacttypes.Where(a => a.Typename == "resource").FirstOrDefault();
+        int lastseq = DbContext.Sectionresources.Where(sr => sr.SectionId == sectionId).OrderByDescending(sr => sr.SequenceNum).FirstOrDefault()?.SequenceNum ?? 0;
+        for (int ix = 0; ix < post.Items?.Length; ix++)
+        {
+            AquiferItem c = post.Items[ix];
             info = await GetContent(c.ContentId, c.ContentType);
+            dynamic? stuff = JsonConvert.DeserializeObject(info);
+
+            string? t = stuff?.grouping.mediaType.Value;
+            string desc = stuff?.localizedName??"";
+            switch (t)
```

Comment:
do you want to set this up to be caseless (i.e. convert t to lower case before checking switch values)?